catalog/views.py

```diff
@@ -27,3 +32,33 @@ def contacts(request):
     template = loader.get_template("catalog/contacts.html")
     context = {"store_contact": store_contact}
     return HttpResponse(template.render(context, request))
+
+
+def product_info(request, product_id):
+    try:
+        product = Product.objects.get(pk = product_id)
```

Comment:
Тут было бы корректне кинуть 404 ошибку, если продукт не найден

```suggestion
        product = get_object_or_404(product_id)
```
Причем ошибку не перехватываем